packages/frameworks/src/frameworks.ts

```diff
@@ -0,0 +1,1605 @@
+    getOutputDirName: async () => 'public',
+  },
+  {
+    name: 'Docusaurus 2',
+    slug: 'docusaurus-2',
+    demo: 'https://docusaurus-2.now-examples.now.sh',
+    logo:
+      'https://raw.githubusercontent.com/vercel/vercel/master/packages/frameworks/logos/docusaurus.svg',
+    tagline:
+      'Docusaurus makes it easy to maintain Open Source documentation websites.',
+    description:
+      'A static Docusaurus site that makes it easy to maintain OSS documentation.',
+    website: 'https://v2.docusaurus.io',
+    detectors: {
+      some: [
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"@docusaurus\\/core":\\s*".+?"[^}]*}',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install` or `npm install`',
+      },
+      buildCommand: {
+        placeholder: '`npm run build` or `docusaurus build`',
+      },
+      devCommand: {
+        value: 'docusaurus start --port $PORT',
+      },
+      outputDirectory: {
+        value: 'build',
+      },
+    },
+    dependency: '@docusaurus/core',
+    devCommand: 'docusaurus start --port $PORT',
+    buildCommand: 'docusaurus build',
+    getOutputDirName: async (dirPrefix: string) => {
+      const base = 'build';
+      try {
+        const location = join(dirPrefix, base);
+        const content = await readdir(location);
+
+        // If there is only one file in it that is a dir we'll use it as dist dir
+        if (content.length === 1 && (await isDir(join(location, content[0])))) {
+          return join(base, content[0]);
+        }
+      } catch (error) {
+        console.error(`Error detecting output directory: `, error);
+      }
+
+      return base;
+    },
+    defaultRoutes: [
+      {
+        src: '^/[^./]+\\.[0-9a-f]{8}\\.(css|js)$',
+        headers: { 'cache-control': 'max-age=31536000, immutable' },
+        continue: true,
+      },
+      {
+        src:
+          '^/assets/images/[^/]+-[0-9a-f]{32}\\.(ico|svg|jpg|jpeg|png|gif|webp)$',
+        headers: { 'cache-control': 'max-age=31536000, immutable' },
+        continue: true,
+      },
+      {
+        src:
+          '^/assets/medias/[^/]+-[0-9a-f]{32}\\.(ogv|wav|mp3|m4a|aac|oga|flac)$',
+        headers: { 'cache-control': 'max-age=31536000, immutable' },
+        continue: true,
+      },
+      {
+        src:
+          '^/assets/files/[^/]+-[0-9a-f]{32}\\.(pdf|doc|docx|xls|xlsx|zip|rar)$',
+        headers: { 'cache-control': 'max-age=31536000, immutable' },
+        continue: true,
+      },
+      {
+        src: '^/ideal-img/[^/]+\\.[0-9a-f]{7}\\.\\d+\\.(png|jpe?g|gif)$',
+        headers: { 'cache-control': 'max-age=31536000, immutable' },
+        continue: true,
+      },
+      {
+        handle: 'filesystem',
+      },
+      {
+        src: '.*',
+        status: 404,
+        dest: '404.html',
+      },
+    ],
+  },
+  {
+    name: 'Docusaurus 1.0',
```

Comment:
```suggestion
    name: 'Docusaurus 1',
```

Was this intentionally changed?